ExchangeSharp/Model/ExchangeOrderRequest.cs

```diff
@@ -70,7 +70,7 @@ public class ExchangeOrderRequest
         /// Not all exchanges will use this dictionary.
         /// These are added after all other parameters and will replace existing properties, such as order type.
         /// </summary>
-        public Dictionary<string, object> ExtraParameters { get; private set; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+        public Dictionary<string, object> ExtraParameters { get; set; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Why does this need a public set?

Reply:
This needs to be set in the ExchangeOrderRequest object being passed to SubmitOrder function from the application code outside this framework, ExtraParameters attribute needs to have public set

Reply:
My preference would be to have everyone use .CopyTo on their dictionary to copy the items to this dictionary, to ensure that any params in there are not blown away. If you want to blow away the parameters already in there, you can clear the dictionary explicitly before calling CopyTo on your dictionary. Anyone else thoughts?

``` myDictionary.CopyTo(orderRequest.ExtraParameters); ```

Reply:
This also avoids null ref exceptions until we implement the new C# 8.X null handling.